Makefile

```diff
@@ -13,10 +13,6 @@ format:
 	cd python-sdk; isort rebuff/ tests/
 	cd python-sdk; black rebuff/ tests/
 
-
-init: init-python-sdk init-server
-	npm install
-
```

Comment:
So this is called in our github workflow:
https://github.com/protectai/rebuff/blob/main/.github/workflows/python_tests.yaml#L27-L28
https://github.com/protectai/rebuff/blob/main/python-sdk/tests/conftest.py#L21-L58

Which stands up the server for python-sdk tests (and subsequently tests ingration with the server -- now pinned to a specific release). Ultimately we should replace this setup with unit testing individual TS functions but for the time being is there a need to remove this?

Reply:
Added it back, although I removed the `npm install`, because the `init-server` step already does that.